pkg/adapter/apiserver/stats_reporter.go

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2019 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package apiserver
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"knative.dev/eventing/pkg/metrics/metricskey"
+	"knative.dev/pkg/metrics"
+)
+
+var (
+	// eventCountM is a counter which records the number of events received
+	// by a Trigger.
```

Comment:
update comment: .... number of events sent by an Importer

Reply:
updated